Libraries/oneMKL/block_lu_decomposition/sample.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Block LU Decomposition",
```

Comment:
**Add a sample GUID** Go here : https://www.guidgenerator.com/ Checkbox 'Uppercase' and 'hyphens' field Copy the value that is like : 825BEB2D-C9FA-4527-8770-9F7D765B65AB

In your sample.json, enter a field called "guid" at the very top and make it :

> "guid" : "825BEB2D-C9FA-4527-8770-9F7D765B65AB"